client/app/utils/PropTypes.js

```diff
@@ -0,0 +1,8 @@
+import { PropTypes } from 'react';
+
+const className = PropTypes.oneOfType([
+  PropTypes.string,
+  PropTypes.objectOf(PropTypes.bool),
+]);
+
+export { className };
```

Comment:
I think you can also add the `export` keyword in front of the `const className` to avoid repetition here.